app/models/compute_resources/foreman/model/openstack.rb

```diff
@@ -35,6 +36,22 @@ def tenant=(name)
       attrs[:tenant] = name
     end
 
+    def allow_ext_net?
+      !(attrs[:allow_ext_net] == 0)
+    end
+
+    def allow_ext_net
+      attrs[:allow_ext_net]
+    end
+
+    def allow_ext_net=(enabled)
+      if ['true', true, '1', 1].include?(enabled)
```

Comment:
Thanks. I wasn't aware of this useful helper method :)
Isn't it better to just:
`attrs[:allow_ext_net] = Foreman::Cast.to_bool(enabled)`